inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/node/doris/DorisDataNodeOperator.java

```diff
@@ -81,4 +86,21 @@ protected void setTargetEntity(DataNodeRequest request, DataNodeEntity targetEnt
         }
     }
 
+    @Override
+    public Boolean testConnection(DataNodeRequest request) {
+        String jdbcUrl = MySQLDataNodeDTO.convertToJdbcurl(request.getUrl());
+        String username = request.getUsername();
+        String password = request.getToken();
+        Preconditions.expectNotBlank(jdbcUrl, ErrorCodeEnum.INVALID_PARAMETER, "connection jdbcUrl cannot be empty");
+        try (Connection ignored = MySQLJdbcUtils.getConnection(jdbcUrl, username, password)) {
+            LOGGER.info("doris connection not null - connection success for jdbcUrl={}, username={}, password={}",
+                    jdbcUrl, username, password);
```

Comment:
Is it ok to output the plain text of the password?

Reply:
This is just a test connection to help users confirm whether the wrong password caused the test to fail, so plain text would be better.